src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocPackageCheckTest.java

```diff
@@ -94,8 +94,8 @@ public void testBoth() throws Exception {
         };
         verify(createChecker(checkConfig),
             getPath("bothfiles" + File.separator + "InputJavadocPackageBothIgnored.java"),
-            getPath("bothfiles"
-            + File.separator + "InputJavadocPackageBothIgnored.java"), expected);
+            getPath("bothfiles" + File.separator + "InputJavadocPackageBothIgnored.java"),
+                expected);
```

Comment:
Please move reformatting to different commit.

Reply:
Done.

Reply:
Resolved.